USING_WITH_PYRIGHT.md

```diff
@@ -0,0 +1,74 @@
+# Pyright and Pylance Together
+
+[Pyright](https://github.com/Microsoft/pyright) is an open source version of a Python type checker that forms the basis for Pylance. Both Pyright and Pylance can be used inside of an IDE to provide a [language server](https://code.visualstudio.com/api/language-extensions/language-server-extension-guide#why-language-server) for Python code.
+
+Pyright can also be used on the [command line](https://github.com/microsoft/pyright/blob/main/docs/command-line.md) to produce json or text output. This can be used in a variety of situations, like a [pre-commit hook](https://github.com/microsoft/pyright/blob/main/docs/ci-integration.md#running-pyright-as-a-pre-commit-hook), or a [github action](https://github.com/jakebailey/pyright-action).
+
+
+However, Pylance and Pyright don't always produce the same results. There are several reasons why this may happen.
+
+## Type stub differences
+
+Pylance comes bundled with a number of stubs, usually found here in the installation:
+
+```
+<pylance-extension-install>/dist/bundled
+<pylance-extension-install>/dist/native-stubs
+```
+
+For Pyright to produce the same errors as Pylance, it needs to use the same stub files. 
+
+This might be done by copying the contents of each of the bundled Pylance stub folders to a new folder. 
+
+You'd then reference that new folder in your [pyrightconfig.json](https://microsoft.github.io/pyright/#/configuration)

```

Comment:
I think we want to list where got them?

https://github.com/microsoft/python-type-stubs and other ones we pull.

Reply:
I think we'd handle that when we create an repository to make this easier. I mean I figure we'll just use git submodule and reference all of the locations that way.